src/compute-pr-actions.ts

```diff
@@ -3,6 +3,7 @@ import { PrInfo, BotError, BotEnsureRemovedFromProject, BotNoPackages } from "./
 import { CIResult } from "./util/CIResult";
 import { ReviewInfo } from "./pr-info";
 import { noNulls, flatten, unique, sameUser } from "./util/util";
+import { userInfo } from "os";

```

Comment:
Likely a mistake

Reply:
Yeah, I caught that in one of the upcoming PRs, so I'll leave this pr as is for now...

(An annoying feature of the Emacs ts thing is that if I mistype and complete a random binding, it will add the import...)